data/json/npcs/refugee_center/surface_refugees/NPC_Jenny_Forcette.json

```diff
@@ -579,6 +579,7 @@
     "id": "MISSION_REFUGEE_Jenny_GET_PNEUMATICS",
     "type": "mission_definition",
     "name": { "str": "Find reference material about air rifles for Jenny." },
+    "description": "Find reference material about air rifles.  <color_yellow>A book that has 'Pneumatic' in its name</color> could be the thing she wants.",
```

Comment:
```suggestion
    "description": "Find reference material about air rifles.  <color_yellow>A book that has 'Pneumatic' in its name</color_yellow> could be the thing she wants.",
```

Reply:
</color> works just fine. dont need to add the yellow again.